query/common/src/test/java/org/codice/ddf/admin/common/fields/test/TestHiddenField.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.common.fields.test;
+
+import org.codice.ddf.admin.common.fields.base.BaseField;
+import org.codice.ddf.admin.common.fields.base.scalar.StringField;
+
+public class TestHiddenField extends StringField {
```

Comment:
❓ Do we need this as a test class anymore? Seems a little overkill now since we no longer have the abstract class.

Reply:
I was using it in the GraphQLTransformation test to see if `*****` is returned instead of whatever is given but I can remove it if you think it's necessary.